curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo-app/64fb1436adef3e145b4c3501.md

```diff
@@ -0,0 +1,386 @@
+---
+id: 64fb1436adef3e145b4c3501
+title: Step 43
+challengeType: 0
+dashedName: step-43
+---
+
+# --description--
+
+Set the `innerText` of the `addOrUpdateTaskBtn` button to `Update Task` and its `arial-label` to `Update Task` too.
```

Comment:
```suggestion
Set the `innerText` of the `addOrUpdateTaskBtn` button to `Update Task` and its `aria-label` to `Update Task` as well.
```

Reply:
@Ksound22 

It looks like this conversation hasn't been resolved yet